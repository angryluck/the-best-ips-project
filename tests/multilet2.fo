fun int multilet_case() =
  let n = read(int); x = n+n; y = x-1
  in write(n * y)

fun int main() =
  let tmp = multilet_case() in
  let tmp = multilet_case() in
  let tmp = multilet_case() in
  let tmp = multilet_case() in
  let tmp = multilet_case() in
  let tmp = multilet_case() in
  let tmp = multilet_case() in
  let tmp = multilet_case() in
  let tmp = multilet_case() in
  multilet_case()
