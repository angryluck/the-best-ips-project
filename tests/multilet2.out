55
3
0
1
6
45
190
210
19900
91
